policies/revault_nothing.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+"""A plugin which never returns any attempt sent to it as needing to be revaulted"""
```

Comment:
nit: Wording is a bit weird (it took a couple of seconds to understand what you meant) but I don't have a better one so :D